sdk/tables/Azure.Data.Tables/src/TableClient.cs

```diff
@@ -1405,6 +1405,10 @@ public virtual Uri GenerateSasUri(TableSasPermissions permissions, DateTimeOffse
         public virtual Uri GenerateSasUri(
             TableSasBuilder builder)
         {
+            if (SharedKeyCredential == null)
```

Comment:
Do the existing tests already cover this scenario, and these changes are just clarifying the message text?

Reply:
Yes - we just fail further into the method when trying to Sign the SAS.